llm/verl/multinode.yaml

```diff
@@ -92,48 +96,77 @@ run: |
     CUDA_DEVICE_MAX_CONNECTIONS: "1"
     HF_HUB_ENABLE_HF_TRANSFER: "1"
   EOF
-  
+
   # Ray cluster configuration
   HEAD_PORT=6385
   DASH_PORT=8280
-  
-  # Function to check if Ray is already running
-  is_ray_alive () {
-    ray status --address="$1:$HEAD_PORT" >/dev/null 2>&1
-  }
-  
+
   if [ "$SKYPILOT_NODE_RANK" == "0" ]; then
     # Head node: prepare data, download model, start Ray head, and submit training job
     echo "Setting up head node..."
-    
+
     # Install additional dependencies for data processing
     uv pip install datasets transformers
-    
+
     # Prepare GSM8K dataset
     python3 examples/data_preprocess/gsm8k.py --local_dir ~/data/gsm8k
-    
+
     # Download model to cache
     python3 -c "import transformers; transformers.pipeline('text-generation', model='Qwen/Qwen2.5-0.5B-Instruct')"
-    
-    # Start Ray head node if not already running
-    if ! is_ray_alive "$head_ip"; then
-      echo "Starting Ray head node..."
-      ray start --head --node-ip-address="$head_ip" \
-          --port $HEAD_PORT --dashboard-port $DASH_PORT \
-          --dashboard-host=0.0.0.0 \
-          --dashboard-agent-listen-port=52366 \
-          --disable-usage-stats \
-          --num-gpus=$SKYPILOT_NUM_GPUS_PER_NODE
-      sleep 10
+  fi
+
+  if [ -f ~/skypilot_templates/ray/start_cluster.sh ]; then
+    export RAY_HEAD_PORT=$HEAD_PORT
+    export RAY_DASHBOARD_PORT=$DASH_PORT
+    export RAY_DASHBOARD_HOST=0.0.0.0
+    export RAY_DASHBOARD_AGENT_LISTEN_PORT=52366
+    export RAY_NODE_IP_ADDRESS="$head_ip"
```

Comment:
Should we call this env var `RAY_HEAD_IP_ADDRESS`?

Reply:
Yes that's more accurate.